### ISSUE TITLE
Non-gui doesn't work 

### ISSUE THREAD

**Issue by karmaical** [DeadBread76/Raid-Toolbox#173]
Whenever I type a number and the specifics it just says it's doing them but does literally nothing

**Comment by hidden-r4d** [comment]
send errors.log, make sure tokens aren't locked, make sure you're not cloudflare banned.

**Comment by karmaical** [comment]
1.How what I send erros.log?
2.How to I find if tokens are locked?
3.how do I check if I'm cloudflare banned and how do I get unbanned?

**Comment by karmaical** [comment]
Uh

**Comment by karmaical** [comment]
Hey uh can anyone help I'm quite perplexed

**Comment by karmaical** [comment]
Ok after doing research I don't think I'm cloudflare banned beacuse I used a vpn and also waited some hours but I have no idea about tokens or errors.log so

**Comment by karmaical** [comment]
> send errors.log, make sure tokens aren't locked, make sure you're not cloudflare banned.

doing more reaserch i cant send an errors log beacuse there is no error it just doesnt do anything but not work

**Comment by hidden-r4d** [comment]
there is a file in the RTB directory named errors.log

**Comment by karmaical** [comment]
How would I find this in the trermux version?

**Comment by arifmamon** [comment]
@karmaical " verify your account before using this token "

**Comment by karmaical** [comment]
> @karmaical " verify your account before using this token "

Yeah I've done that years ago matter of fact I have 2fa enabled I'll disable it and try
